src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, onTabSelected, currentTab }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map((tab) => {
+          const isCurrent = tab === currentTab;
```

Comment:
you can not compare objects...compare their id

` tab.id === currentTab.id`

also update `isCurrent` to `isCurrentTabSelected`